.github/CODEOWNERS

```
@@ -1 +1 @@
-.github/CODEOWNERS @sonarsource/sonarcloud
+.github/CODEOWNERS @sonarsource/analysis-experience
```

```suggestion
.github/CODEOWNERS @sonarsource/analysis-experience-squad
```

The Analysis XP squad handle on Github is `@sonarsource/analysis-experience-squad`